board.py

```diff
@@ -1,33 +1,90 @@
 class Board:
-    board_state = []
+    def __init__(self):
+        self.board_state = []
 
     def new_game(self, numrows=3):
         board_area = numrows*numrows
-        new_board = [" "] * board_area
+        new_board = [""] * board_area
```

Comment:
Why an array of strings? Did you consider other ways of modeling the board data? What about this data structure makes it best (if it is the best?)

Reply:
I chose to use an array of strings because I wasn't that familiar with python dictionaries. I could have used a dictionary and ended up with a board like this {1: "1", 2: "2", 3: "3" etc....}. 

The array (list) seemed fairly simple because I could also use the indices. I attempted to have a "None" value but couldn't build a board string for the terminal. I had to go back to using a empty string.